appscale/tools/parse_args.py

```diff
@@ -13,6 +13,10 @@
 
 
 # AppScale-specific imports
+try:
+  from agents.azure_agent import AzureAgent
```

Comment:
This fails in OCD while creating a Euca image because at that point the tools build is not run where the pip install --pre azure is present. This was the same issue we were facing in the azure-agent branch where the setup.py is run before and the build later.